opal/threads/wait_sync.c

```diff
@@ -93,8 +93,10 @@ int sync_wait_mt(ompi_wait_sync_t *sync)
     /* In case I am the progress manager, pass the duties on */
     if( sync == wait_sync_list ) {
         wait_sync_list = (sync == sync->next) ? NULL : sync->next;
-        if( NULL != wait_sync_list )
+        if( NULL != wait_sync_list ){
+            /* This is a possible placement for a progress switch counter */
```

Comment:
The counter driver code has been moved up to the OMPI level, so this is no longer a viable counter.  Perhaps this could still be an MPI_T pvar.